### PR TITLE
doc / bitmart connector dev

### PR DESCRIPTION
**Pull request by glenntomas**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

- Add BitMart connector

image

**Tests performed by the developer**:

**Tips for QA testing**:

### CODE REVIEW

File: documentation/docs/exchanges/bitmart.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## 📁 [Connector folder](https://github.com/CoinAlpha/hummingbot/tree/development/hummingbot/connector/exchange/bitmart)
+
+## ℹ️ Exchange Info
+
+**BitMart**
+[Website](https://www.bitmart.com/) | [CoinMarketCap](https://coinmarketcap.com/exchanges/bitmart/) | [CoinGecko](https://www.coingecko.com/en/exchanges/bitmart)
+
+- API docs: https://developer-pro.bitmart.com/en/
+- Transaction fees: https://www.bitmart.com/fee/en
+- Minimum order size: Equivalent value of 5 USDT
```
Comment (mifeng) [inline documentation/docs/exchanges/bitmart.md:17]: If we don't have a source for the minimum order, then leave it blank.

File: documentation/docs/exchanges/bitmart.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Creating API keys: https://support.bmx.fund/hc/en-us/articles/360016076854-How-to-Create-An-API
```
Comment (mifeng) [inline documentation/docs/exchanges/bitmart.md:18]: Ideally, we only source pages that are on the Bitmart website. However, if there's no page in their help center or support, external sources are OK.

**Review by mifeng — CHANGES_REQUESTED** [review]

File: documentation/docs/exchanges/bitmart.md (hunk 1 above)
Comment (glenntomas) [reply documentation/docs/exchanges/bitmart.md]: Removed the minimum order size, I can't find any link that shows this on BitMart.

File: documentation/docs/exchanges/bitmart.md (hunk 2 above)
Comment (glenntomas) [reply documentation/docs/exchanges/bitmart.md:18]: Copy on this sir. I found a link on their help center regarding fees, I change the link to that one.

**Review by mifeng — APPROVED** [review]

**Pull request merged**